odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -1215,7 +1215,28 @@ def _mocked_check_access_rights(*args, **kwargs):
         self.env.invalidate_all()
         with patch.object(MultiTag, 'check_access_rights', side_effect=_mocked_check_access_rights):
             values = self.message_1.read(['attributes'])[0]['attributes'][0]
-        self.assertEqual(values['value'], (tag.id, 'No Access'))
+        self.assertEqual(values['value'], (tag.id, None))
+
+        # a user read a properties with a many2many and he doesn't have access to all records
```

Comment:
Would make a dedicated unit test for m2m filtering (better to keep tests small and to the point)